x/dogfood/keeper/impl_sdk.go

```diff
@@ -103,7 +96,7 @@ func (k Keeper) SlashWithInfractionReason(
 	}
 	// TODO(mm): add list of assets to be slashed (and not just all of them).
 	// based on yet to be finalized slashing design.
-	return k.slashingKeeper.SlashWithInfractionReason(
+	return k.operatorKeeper.SlashWithInfractionReason(
 		ctx, accAddress, infractionHeight,
 		power, slashFactor, infraction,
```

Comment:
lack of validation of legitimacy of some parameters (slashFactor --> slashProportion), need we  make sure it is in a reasonable  range ？

Reply:
The `slashFactor` is a parameter of the slashing module of cosmos-sdk. It should have been verified when configuring it.